src/org/exist/xquery/functions/fn/FunUnparsedText.java

```diff
@@ -28,6 +29,7 @@
 import org.exist.xmldb.XmldbURI;
 import org.exist.xquery.*;
 import org.exist.xquery.value.*;
+import org.junit.runner.RunWith;
```

Comment:
junit?

Reply:
Oops, that went into the wrong class. I have fixed that now.